docs/imaging/sync.rst

```diff
@@ -4,31 +4,117 @@
+Each of these implementation is then inserted into a very generalised schema, which maps the time data from each time series into a single temporal frame of reference. This generalised approach has both benefits and drawbacks: the primary benefit is that it can be extended to new types of data - and new hardware - very easily. The primary drawback is that it adds complexity and overhead to common operations, and that the users need to be aware of that complexity. 
 
+Synchronisation data is stored in the ``Sync`` table and its sub-tables. 
 
-.. figure:: /_static/imaging/sync_tables.PNG
+.. figure:: /_static/imaging/sync/sync_tables.PNG
    :alt: Synchronisation section of the imaging schema
 
+Each Session is generated by a **Setup** (:ref:`Imaging Terminology Setup`), which is tracked in the ``Setup`` table. Each entry in the ``Setup`` table has a list of the types of synchronisation data (and how to interpret it) that might be generated in the table ``Setup.Sync``.::
+
+  Setup.Sync & {"setup_name":"Mini Blue", "setup_timestamp":"2020-08-20 00:00:00"}
+
+.. figure:: /_static/imaging/sync/setup-sync-table.png
+   :alt: Contents of the ``Setup.Sync`` table
+
+
+When a new Session is ingested, the synchronisation data is inserted into the ``Sync`` table and its part tables. Each Session has one (or more) entries in the ``Sync`` table, corresponding to a single time series, from a single instrument, with each time series being identified by a ``sync_name``.
+
+The actual timestamps themselves are stored in the column ``sync_data`` of the ``Sync`` table. The values are stored in *samples*. The sample rate (samples per second) is also stored, in column ``sample_rate``, if you wish to convert into time periods in seconds. 
+
+
+Sync Names and Generic Names
+--------------------------------
+
+The ``Setup.Sync`` entries have two related attributes:
+
+* ``sync_name``
+* ``generic_name``
+
+The most important attribute here is the ``generic_name``. This defines how further downstream tables identfy *which* Sync stream they should be looking for, to pair time-series data with its timestamps. The main ``generic_names`` are as follows:
+
+* ``frames_imaging`` : timestamps for two-photon imaging frames
```

Comment:
..., defined as the start time of scanning on each frame. For multi plane recordings this resolves to the start time of scanning of each volume (=stack), such that the number of data points saved here matches the number of distinct timepoints recorded for single cells (i.e. number of data points in sync equals the number of data points in imaging analysis output like suite2p for every cell).